tarot/urls.py

```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     path("gameInfo/", views.my_view, name="game_info"),
+    path("back/", views.my_view, name="card_back")
```

Comment:
what is `back` mean? could you seperated to another view like `views/back.py`?

Reply:
We used `back/` to conform to the URL format specified in the API we wrote to Figma.

Reply:
Okay. How about seperate?

Reply:
I understand what you're saying, and since we're working on something after this.
Can I commit this contents in next pull request?